pkg/asset/machines/master.go

```diff
@@ -888,3 +899,18 @@ func createAssetFiles(objects []interface{}, fileName string) ([]*asset.File, er
 
 	return assetFiles, nil
 }
+
+// supportedSingleNodePlatform indicates if the IPI Installer can be used to install SNO on
+// a platform.
+func supportedSingleNodePlatform(bootstrapInPlace bool, platformName string) bool {
+	switch platformName {
+	case awstypes.Name, gcptypes.Name, azuretypes.Name, powervstypes.Name, baremetaltypes.Name:
```

Comment:
Baremetal IPI does not support SNO. It only works with platform None, which is handled below.